environment/galayou_Y4-t31l-atbm6012bx.txt

```diff
@@ -0,0 +1,23 @@
+day_night_max=15000
+day_night_min=500
+disable_eth=true
+enable_updates=true
+gpio_button=14
+gpio_default=7O 8o 9O 10o 11o 14i 16i 17o 18O 49o 52o 53o 57o 58o 59o 61o 62o 63o 64O
+gpio_ir940=11
+gpio_ircut=57 58
+gpio_led_b=9
```

Comment:
can you add a suffix to the gpios? O=gpio high to turn on or o=gpio low to turn on? for both LEDs please.

Reply:
Is it needed, while it is set in the gpio_defaults?